src/ansys/fluent/core/launcher/pyfluent_enums.py

```diff
@@ -17,48 +17,99 @@
 import ansys.platform.instancemanagement as pypim
 
 
-class LaunchMode(Enum):
-    """Enumerates over supported Fluent launch modes."""
+@total_ordering
+class FluentEnum(Enum):
+    """Provides the base class for Fluent-related enums.
 
-    STANDALONE = 1
-    PIM = 2
-    CONTAINER = 3
-    SLURM = 4
+    Accepts lowercase member names as values and supports comparison operators.
+    """
 
+    def _get_enum_map(self):
+        _fl_val_map = {}
+        return _fl_val_map
 
-class FluentMode(Enum):
-    """Enumerates over supported Fluent modes."""
+    def get_fl_value(self):
+        """Returns the fluent value of the enum."""
```

Comment:
I would suggest being more verbose than just using "fl" for "fluent", makes the code more readable and avoids confusion in the future:
```suggestion
    def get_fluent_value(self):
        """Returns the Fluent value of the enum."""
```
But what do we mean by this? This value isn't anything that is being sent to or used by Fluent. These are being used by PyFluent, for example `Meshing` session object for `FluentMode.MESHING`